oshi-core-java24/src/main/java/oshi/hardware/linux/LinuxHardwareAbstractionLayer.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright 2025 The OSHI Project Contributors
+ * SPDX-License-Identifier: MIT
+ */
+package oshi.hardware.linux;
+
+import oshi.hardware.HardwareAbstractionLayer;
+
+public class LinuxHardwareAbstractionLayer implements HardwareAbstractionLayer {
```

Comment:
Whether we copy these over or keep our own copy may be dependent on how different the implementation is.  For now, the empty class is useless if we're copying over it.